Source/ACE/Entity/Player.cs

```diff
@@ -63,20 +62,7 @@ public enum ConsumableBuffType : uint
         /// </summary>
         private Dictionary<ObjectGuid, WorldObject> interactiveWorldObjects = new Dictionary<ObjectGuid, WorldObject>();
 
-        /// <summary>
```

Comment:
what happened to these 3 properties?  were they not needed?

Reply:
They were stuff I had added 6 months ago and had no references.    I think they were added before I knew what was going on.